navigation.yaml

```diff
@@ -643,3 +643,5 @@ pricing:
       path: /pricing
     - title: Ubuntu Advantage Infrastructure
```

Comment:
Can you make this Support for now? see the Readme in the folder (link on the spreadsheet)

Reply:
Can't find a comment about this? Did you still want the URL to be /pricing/infra?